core/src/main/java/com/google/cloud/sql/CredentialFactory.java

```diff
@@ -20,7 +20,7 @@
 
 /**
  * Factory for creating {@link Credential}s for interaction with Cloud SQL Admin API.
- */
+ * */
```

Comment:
```suggestion
 */
```

google-java-format should have caught this.